tests/L0/run_optimizers/test_distributed_fused_lamb.py

```diff
@@ -0,0 +1,123 @@
+import os
```

Comment:
could you put this file in apex/contrib/tests/optimizers for the consistency with distributed_fused_adam?